recipe/meta.yaml

```diff
@@ -17,15 +17,15 @@ requirements:
     - toolchain
     - cmake >=3.2
     - zlib 1.2.11
-    - bzip2 1.0
+    - bzip2 1.0.6
     - zstd 1.3.2
-    - lz4-c 1.8.1
+    - lz4-c 1.8.0
```

Comment:
We'd want `1.8.1|1.8.1.*` I think, except that `1.8.1` isn't actually on conda-forge yet. Sending a PR shortly.